components/FilesTable/FilesTable.vue

```diff
@@ -192,7 +192,6 @@
                 </sparc-tooltip>
               </div>
               <div
-                v-if="hasOsparcViewer(scope)"
```

Comment:
So we want to show this always, even if there is no viewer available?

Reply:
Yes, now it is possible to open every file type for custom processing in oSparc